File: game_server/logic_server/src/main/java/chr/fotmation/Formation.java
package chr.fotmation;

/**
 * Created by think on 2017/9/12.
 */
public class Formation {
	private long heroId;

	private long[] equipId = new long[6];

	private long petId;

	private long horseId;

	private long mingjiangId;

	public Formation(long heroId) {
		this.heroId = heroId;
	}

	public long getMingjiangId() {
		return mingjiangId;
	}

	public void setMingjiangId(long mingjiangId) {
		this.mingjiangId = mingjiangId;
	}

	public long getHeroId() {
		return heroId;
	}

	public void setHeroId(long heroId) {
		this.heroId = heroId;
	}

	public long[] getEquipId() {
		return equipId;
	}

	public void setEquipId(long[] equipId) {
		this.equipId = equipId;
	}

	public long getPetId() {
		return petId;
	}

	public void setPetId(long petId) {
		this.petId = petId;
	}

	public long getHorseId() {
		return horseId;
	}

	public void setHorseId(long horseId) {
		this.horseId = horseId;
	}
}

File: godsg/Assets/client_code/logic/entity/Formation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

class Formation
{
    private long heroId;

    private long[] equipList;

    private long petId;

    private long horseId;

    private long mingjiangId;

    public Formation()
    {
        this.heroId = 0;
        this.equipList = new long[6];
        this.petId = 0;
        this.horseId = 0;
        this.mingjiangId = 0;
    }

    public long HeroId
    {
        get
        {
            return heroId;
        }

        set
        {
            heroId = value;
        }
    }

    public long[] EquipList
    {
        get
        {
            return equipList;
        }

        set
        {
            equipList = value;
        }
    }

    public long PetId
    {
        get
        {
            return petId;
        }

        set
        {
            petId = value;
        }
    }

    public long HorseId
    {
        get
        {
            return horseId;
        }

        set
        {
            horseId = value;
        }
    }

    public long MingjiangId
    {
        get
        {
            return mingjiangId;
        }

        set
        {
            mingjiangId = value;
        }
    }
}
